views/error/not_found.blade.php

```diff
@@ -4,6 +4,12 @@
   <p>
     {{ $message }}
   </p>
+  <p>
+    <form class="form" method="GET" action="/">
```

Comment:
We cannot use `/` here for the action as it won't work with subfolders.
And is there a reason why the `form` has a class? It's not used anywhere, and styles only come from the inline `style` tag.

```suggestion
    <form class="form" method="GET" action="{{ $app->url() }}">
```